src/cls/IPM/General/SemanticVersionExpression/Comparator.cls

```diff
@@ -42,6 +42,7 @@ ClassMethod FromString(
 	pExpr As %String,
 	Output pComparator As %IPM.General.SemanticVersionExpression.Comparator) As %Status
 {
+  set pExpr = $zstrip(pExpr, "*", "()")
```

Comment:
Parsing should be updated to handle parentheses instead of stripping them away everywhere IMO